<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditorDemo</class>
 <widget class="QWidget" name="TextEditorDemo">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="bold_btn">
       <property name="text">
        <string>Bold</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="italic_btn">
       <property name="text">
        <string>Italic</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="underline_btn">
       <property name="text">
        <string>Underline</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="text_edit">
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>


